charts/fluentd-elasticsearch/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: fluentd-elasticsearch
-version: 9.3.2
+version: 9.3.3
```

Comment:
This is a new feature so Version should be 9.4.0

Reply:
done